packages/perseus/src/widgets/interactive-graphs/graphs/angle.tsx

```diff
@@ -95,6 +97,64 @@ function AngleGraph(props: AngleGraphProps) {
         showAngles: showAngles || false, // Whether to show the angle or not
     };
 
+    const angleMeasure = getClockwiseAngle(
+        [endPoints[0], centerPoint, endPoints[1]],
+        allowReflexAngles,
+    );
```

Comment:
I think you might have to trim this angle measure so it's only integers or maybe only shows 1 or 2 significant figures. I think you can use `srFormatNumber()` for this here too.

<img width="834" alt="image" src="https://github.com/user-attachments/assets/42a38987-f93f-446e-871d-dbe256e75d1b" />